tests/python/test_matrix.py

```diff
@@ -1195,6 +1195,23 @@ def verify(x):
     _test_field_and_ndarray(field, ndarray, func, verify)
 
 
+@test_utils.test(arch=[ti.cuda, ti.cpu],
+                 real_matrix=True,
+                 real_matrix_scalarize=True,
+                 debug=True)
```

Comment:
```suggestion
@test_utils.test()
```
This check should apply to all backends.